src/screens/WalletConnect/WalletConnectPinConfirm.js

```diff
@@ -164,6 +184,23 @@ class WalletConnectPinConfirmScreeen extends React.Component<Props, State> {
     );
   };
 
+  handleSignTypedData = async (request: CallRequest, wallet: Object) => {
+    const { approveCallRequest, rejectCallRequest } = this.props;
+    try {
+      const message = request.params[1]; // eslint-disable-line
+      const result = await signTypedData(message, wallet);
+      await approveCallRequest(request.callId, result);
+    } catch (error) {
+      await rejectCallRequest(request.callId, error.toString());
+    }
```

Comment:
Shouldn't we log the error?

Reply:
Not sure because this can be simple reject on dapp end, but it also includes Private Key in signing  process which we don't want to expose by any mean 🤔